design-library/source/_patterns/04-pages/01-one-sidebar/one-sidebar.twig

```diff
@@ -0,0 +1,30 @@
+<div class="page-wrapper">
```

Comment:
@bixgomez did you rename my existing pattern? if so, why? this might break my drupal connectivity

Reply:
@nerdstein : Yes, I did -- I felt that simply naming the page "sidebar" was insufficiently descriptive.  My intent is to repair drupal connectivity to suit.

Reply:
ok